modules/apps/27-interchain-accounts/keeper/account_test.go

```diff
@@ -67,3 +68,74 @@ func (suite *KeeperTestSuite) TestInitInterchainAccount() {
 		})
 	}
 }
+
+func (suite *KeeperTestSuite) TestInitChannel() {
+	var (
+		path   *ibctesting.Path
+		portID string
+	)
+
+	testCases := []struct {
+		name     string
+		malleate func()
+		expPass  bool
+	}{
+		{
+			"success", func() {}, true,
+		},
+		{
+			"port is not bound", func() {
+				portID = "invalid-portID"
+			}, false,
+		},
+	}
+
+	for _, tc := range testCases {
+		tc := tc
+
+		suite.Run(tc.name, func() {
+			suite.SetupTest() // reset
+			path = NewICAPath(suite.chainA, suite.chainB)
+			suite.coordinator.SetupConnections(path)
+			portID, _ = types.GeneratePortID(TestOwnerAddress, path.EndpointA.ConnectionID, path.EndpointA.Counterparty.ConnectionID)
+
+			// bind port and claim capability
+			cap := suite.chainA.GetSimApp().ICAKeeper.BindPort(suite.chainA.GetContext(), portID)
```

Comment:
Also not entirely sure what the best method for testing this out is. I initially made a call to `InitInterchainAccount` and finished the handshake, and then tried to call `InitChannel` and finish a second handshake, but I ran into verification issues. This is probs overkill anyway. 

Basically, I'm just binding the port fresh here and then checking that we can finish a handshake.